lib/index.js

```diff
@@ -30,15 +30,22 @@ class EvervaultClient {
         Buffer.from(apiKey.slice(3), 'base64')
       );
     }
+    let curve;
+    if (!options.curve || !this.config.encryption[options.curve]) {
+      curve = 'secp256k1'; //default to old curve
+    } else {
+      curve = options.curve;
+    }
 
     this.config = Config(apiKey);
+    this.curve = curve;
     this.retry = options.retry;
     this.http = Http(this.config.http);
-    this.crypto = Crypto(this.config.encryption, this.http);
+    this.crypto = Crypto(this.config.encryption[curve], this.http);
```

Comment:
do we have any validation on the curves? Would also be good to expose as a constant from the library